code/modules/atmospherics/machinery/components/binary_devices/volume_pump.dm

```diff
@@ -115,8 +116,8 @@
 /obj/machinery/atmospherics/components/binary/volume_pump/ui_data()
 	var/data = list()
 	data["on"] = on
-	data["rate"] = round(transfer_rate)
-	data["max_rate"] = round(MAX_TRANSFER_RATE)
+	data["rate"] = round(transfer_rate SECONDS / SSair.wait)
+	data["max_rate"] = round(MAX_TRANSFER_RATE SECONDS / SSair.wait)
```

Comment:
do not like this. would rather we do this on display rather then when we send it over

Reply:
In the tgui file? Why's that?

Reply:
this sort of thing is asking to introduce rounding errors. I'd like to keep the scaling to the display, rather then the actual value